src/jquery.mockjax.js

```diff
@@ -667,6 +667,22 @@
 		origSettings.urlParams = paramValues;
 	}
 
+	/**
+	 * Clears handlers that mock given url
+	 * @param url
+	 * @returns {Array}
+	 */
+	function clearByUrl(url) {
+		var results = [];
+		for (var i=0, len=mockHandlers.length; i<len; i++) {
```

Comment:
Can you update the variable declaration to match our other function style? We like to do single `var` statements at the top to avoid issues with hoisting:

```js
function clearByUrl(url) {
	var i, len,
		results = [];
	for (i=0, len=.......) {
	...
}
```